kotsadm/pkg/supportbundle/supportbundle.go

```diff
@@ -2,38 +2,20 @@ package supportbundle
 
 import (
 	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"net/url"
 	"os"
 	"path/filepath"
 	"time"
 
-	"github.com/aws/aws-sdk-go/aws"
-	awssession "github.com/aws/aws-sdk-go/aws/session"
-	"github.com/aws/aws-sdk-go/service/s3"
+	"github.com/mholt/archiver"
 	"github.com/pkg/errors"
 	"github.com/replicatedhq/kots/kotsadm/pkg/persistence"
-	kotss3 "github.com/replicatedhq/kots/kotsadm/pkg/s3"
 	"github.com/replicatedhq/kots/kotsadm/pkg/supportbundle/types"
 	"github.com/segmentio/ksuid"
 )
 
-func SetBundleAnalysis(id string, insights []byte) error {
```

Comment:
was this never called?

Reply:
moved to analysis.go.  this file was big, splitting it up since i added a lot more to it

Reply:
🤦 the downsides of marking files as 'read' as you go is that you can't cmd+f them